src/main/kotlin/com/autonomousapps/internal/utils/gradleStrings.kt

```diff
@@ -231,8 +234,8 @@ internal fun Dependency.targetGradleVariantIdentification()  = when(this) {
  * attributes (and the same capability). Hence, we do not need to distinguish variants based on attributes (only by
  * capabilities).
  */
-internal fun ResolvedVariantResult.toGradleVariantIdentification() = GradleVariantIdentification(
-  capabilities.map { it.toGA() }.toSet(), emptyMap()
+internal fun ResolvedVariantResult?.toGradleVariantIdentification() = GradleVariantIdentification(
+  this?.capabilities?.map { it.toGA() }?.toSet() ?: emptySet(), emptyMap()
```

Comment:
Not a big deal, but just FYI there is a `Set<T>?.orEmpty()` function that is a bit more concise than `?:`.